wp-content/plugins/wdgrestapi/libs/gateways/lemonway.php

```diff
@@ -138,6 +143,29 @@ private function get_last_error_message() {
 /**
 * GESTION WALLETS
 */
+	/**
+	 * @param int $wallet_id
+	 */
+	public function get_wallet_details( $wallet_id ) {
+		if ( empty( $wallet_id ) ) return FALSE;
+		
+		$result = FALSE;
+
+		if ( !empty( $wallet_id ) ) {
+			$param_list = array( 'wallet' => $wallet_id );
+			$result = $this->call( 'GetWalletDetails', $param_list );
+		}
```

Comment:
@emilschn code redondant avec ce qui est fait au-dessus

		if ( empty( $wallet_id ) ) {
return FALSE;
}else{
$result = FALSE;
			$param_list = array( 'wallet' => $wallet_id );
			$result = $this->call( 'GetWalletDetails', $param_list );
		}